server/model/paginated-results.js

```diff
@@ -0,0 +1,12 @@
+import type {Article} from './article';
+import type {ExhibitionPromo} from './exhibition-promo';
+import type {Pagination} from './pagination';
+
+export type PaginatedResults = {|
+  currentPage: number,
+  results: List<Article | ExhibitionPromo>,
+  pageSize: number,
+  totalResults: number,
+  totalPages: number,
+  pagination: Pagination
+|};
```

Comment:
So what logic are we using between the separation of `content-model` and `model`?
Assuming none and we use `model` for now?